Carthage Firebase.framework does not contain bitcode.

When installed through Carthage, `Firebase.framework/Firebase' does not contain bitcode.`

#### My Environment
 
  * Xcode version: 9.2
  * Firebase SDK version: 4.9.0 (Carthage)
  * Firebase Component: Core, Analytics, Messaging
  * Component version: 4.9.0

#### Steps to reproduce:

1. Install Firebase from Carthage following the steps outlined in the [guide](https://github.com/firebase/firebase-ios-sdk/blob/master/Carthage.md). 
2. Try and build to a physical device or create an Archive 
3. Encounter error:
```
ld: '........../Carthage/Build/iOS/Firebase.framework/Firebase' does not contain bitcode. You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. file '........./Carthage/Build/iOS/Firebase.framework/Firebase' for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

#### Additional Details:
- This is only a problem with Firebase.framework (the other Carthage-installed dependencies I have in the same project do not have this issue)

Hello!

I run into same issue, so I also deleted Firebase.framework from the Link Binary With Libraries Build Phase. 

I am still getting errors like:
`Ld .../Library/Developer/Xcode/DerivedData/CarthageFirebaseTest-gcmfvqeteiovpadbjrrhdofxjsbi/Build/Products/Debug-iphoneos/CarthageFirebaseTest.app/CarthageFirebaseTest normal arm64
    cd .../carthage_firebase_test/CarthageFirebaseTest`

`Undefined symbols for architecture arm64:
  "___cxa_guard_acquire", referenced from:
      leveldb::crc32c::Extend(unsigned int, char const*, unsigned long) in leveldb-library(crc32c.o)
  "___cxa_guard_abort", referenced from:
      leveldb::crc32c::Extend(unsigned int, char const*, unsigned long) in leveldb-library(crc32c.o) ...`

I tried to add additional Linker Flags, which were suggested in another threads and also added libraries like "libicucore.dylib" but it didnt helped. I am only getting different errors.

I am using Xcode 9.2 and macOS Sierra 10.12.6.

 @paulb777, Do you have any suggestion to fix this problem?

--- a/Carthage.md
+++ b/Carthage.md
@@ -48,6 +48,7 @@ binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseStorageBinary.jso
     they're added to the right build target(s).
 - Add the -ObjC flag to "Other Linker Flags".
 - Make sure that the build target(s) includes your project's `GoogleService-Info.plist`.
+- [Delete Firebase.framework from the Link Binary With Libraries Build Phase](https://github.com/firebase/firebase-ios-sdk/issues/911#issuecomment-372455235).
 
 ## Versioning
 

